templates/ts-apollo-fullstack/client/src/generated-types.tsx

```diff
@@ -14,22 +14,22 @@ export type Scalars = {
 export type Comment = {
```

Comment:
Do we actually need this file? We can use typescript to load queries

Reply:
Since the queries are written in there,we need it